.github/workflows/add-to-project-dependabot.yml

```diff
@@ -0,0 +1,16 @@
+name: "Project triage: Dependabot"
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-to-project-dependabot:
+    runs-on: ubuntu-latest
+    # see https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#responding-to-events
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/cal-itp/projects/${{ secrets.GH_PROJECT }}
+          github-token: ${{ secrets.GH_PROJECTS_TOKEN }}
```

Comment:
Do we need to add this as a [Dependabot secret](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#accessing-secrets)?

Reply:
Oh well that's a bummer. I guess we need both of these as Dependabot secrets!

Reply:
I wonder if `github.token` would work? I came across it earlier today: https://github.com/actions/add-to-project/issues/93#issuecomment-1122722536

How to test this though? Would you have to test in a fork?

Reply:
Well `secrets.GH_PROJECT` was just a hedge against hardcoding the project ID in the workflow 😁 I didn't realize you had been looking into this more!

But I think we can just keep it straightforward and add Dependabot secrets for each of these, one at the org level (`GH_PROJECTS_TOKEN`) and one in this repo (`GH_PROJECT`)